src/backend/marsha/core/views.py

```diff
@@ -423,9 +423,22 @@ def _get_resource(self):
 
     def _init_context(self):
         """Extract LTI information (and verifies the request) before data process."""
+        resource_id = self.kwargs.get("uuid")
+        if not resource_id:
+            try:
+                resource_id = self.model.objects.values_list("id", flat=True).get(
+                    lti_id=self.request.POST.get("resource_link_id"),
+                    playlist__lti_id=self.request.POST.get("context_id"),
```

Comment:
This should be tried in a portability context. The video exists and we try to access it from another playlist.

Reply:
Maybe I'm wrong, but for this case, regular url (with resource id) would be used, no ?